.github/workflows/docker-cron.yaml

```diff
@@ -90,4 +90,4 @@ jobs:
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} openfe --version
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} python -c "import gufe; print(gufe.__version__)"
           docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs gufe -v
-          docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs openfecli --pyargs openfe -v
+          docker run --rm ${{ steps.fqirp.outputs.FQIRP }} pytest --pyargs openfe --pyargs openfecli -v
```

Comment:
So there's some kind of fixture contention right? Do we want to raise a separate issue for this?

Reply:
Yes I need to, I am sure we would get the same error if we ran our test like this on CI (using pyargs and calling`openfe` and `openfecli`)

Reply:
Issue raised here: https://github.com/OpenFreeEnergy/openfe/issues/643
I was able to narrow it down to `test_main_log` test for `openfecli`